README.md

```diff
@@ -61,6 +61,6 @@ If you're a student, I strongly recommend applying for the [Github Student Devel
 
 ### Domains
 * [Namcheap](https://www.namecheap.com/promos/amazing98s/): Various TLD's for <$1
```

Comment:
You could also update the typo here, `[Namcheap]`, while you're at it 👍

Reply:
@marmichalski done